met-etl/container/user_code/Dockerfile

```
@@ -1,12 +1,12 @@
-FROM python:3.12.2-bullseye
+FROM python:3.12-alpine
 
-ARG DAGSTER_VERSION=0.23.15
+ARG DAGSTER_VERSION=0.26.11
```

A humble suggestion while we're updating this file, to avoid any future confusion
```suggestion
ARG DAGSTER_LIBRARY_VERSION=0.26.11
```

---

That's definitely clearer. Updated!